openedx/core/djangoapps/credit/tasks.py

```diff
@@ -11,12 +11,12 @@
 from celery.utils.log import get_task_logger
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey
+from opaque_keys.edx.keys import UsageKey
 
 from .api import set_credit_requirements
```

Comment:
Can you please replace this import with full path:
`from openedx.core.djangoapps.credit.api import set_credit_requirements`